main.nf

```diff
@@ -24,55 +24,77 @@ def helpMessage() {
-
-    Presets:
-      --pico                        Sets trimming and standedness settings for the SMARTer Stranded Total RNA-Seq Kit - Pico Input kit. Equivalent to: --forward_stranded --clip_r1 3 --three_prime_clip_r2 3
-      --fcExtraAttributes           Define which extra parameters should also be included in featureCounts (default: gene_names)
-      --fcGroupFeatures             Define the attribute type used to group features. (default: 'gene_name')
-      --fcGroupFeaturesType         Define the type attribute used to group features based on the group attribute (default: 'gene_biotype')
+      --three_prime_clip_r2 [int]   Instructs Trim Galore to remove bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed
+      --trim_nextseq [int]          Instructs Trim Galore to apply the --nextseq=X option, to trim based on quality after removing poly-G tails
+      --pico                        Sets trimming and standedness settings for the SMARTer Stranded Total RNA-Seq Kit - Pico Input kit. Equivalent to: --forwardStranded --clip_r1 3 --three_prime_clip_r2 3
+      --saveTrimmed                 Save trimmed FastQ file intermediates
 
-    Other options:
+    Ribosomal RNA removal:
+      --remove_rRNA                 Removes rRNA using SortMeRNA
+      --save_nonrRNA_reads          Save FastQ file intermediates after removing rRNA
+      --rRNA_database_manifest      Path to file that contains file paths for rRNA databases, optional
+
+    Alignment:
+      --aligner                     Specifies the aligner to use (available are: 'hisat2', 'star')
+      --pseudo_aligner              Specifies the pseudo aligner to use (available are: 'salmon'). Runs in addition to `--aligner`
+      --stringTieIgnoreGTF          Perform reference-guided de novo assembly of transcripts using StringTie i.e. dont restrict to those in GTF file
+      --seq_center                  Add sequencing center in @RG line of output BAM header
+      --saveAlignedIntermediates    Save the BAM files from the aligment step - not done by default
+      --saveUnaligned               Save unaligned reads from either STAR, HISAT2 or Salmon to extra output files.
+      --skipAlignment               Skip alignment altogether (usually in favor of pseudoalignment)
+
+    Read Counting:
+      --fc_extra_attributes         Define which extra parameters should also be included in featureCounts (default: 'gene_name')
+      --fc_group_features           Define the attribute type used to group features. (default: 'gene_id')
+      --fc_count_type               Define the type used to assign reads. (default: 'exon')
+      --fc_group_features_type      Define the type attribute used to group features based on the group attribute (default: 'gene_biotype')
+
+    QC:
+      --skipQC                      Skip all QC steps apart from MultiQC
```

Comment:
Why is skipQC not skipping MultiQC?
We took a different approach within Sarek, we should speak more about this kind of params as well

Reply:
Most users of rnaseq would still want the pipeline to produce a report of some form - thats the main reason for it 👍

Reply:
I see, then I guess that the approach we have in sarek could be nice for this case.
We do `--skipQC tool1,tool2,tool3` in the same way we do `--tools tool1,tool2`
But we can think of this for another release, and discuss it at a nf-core level too.

Reply:
Yup, we should have a discussion maybe for the template then and introduce this on an nf-core level? I quite like your approach and think that could be something that works for almost all pipelines with potentially optional processes 👍

Reply:
Mind making an issue as this is your idea? :-D